src/components/v5/shared/CopyWallet/CopyWallet.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import Button from '~v5/shared/Button';
+import { formatText } from '~utils/intl';
+import Icon from '~shared/Icon';
+import { CopyWalletProps } from './types';
+import { splitWalletAddress } from '~utils/splitWalletAddress';
+import { useMobile } from '~hooks';
+
+const displayName = 'v5.CopyWallet';
+
+const CopyWallet: FC<CopyWalletProps> = ({
```

Comment:
Please make this component reusable - don't use `walletAddress` here, just pass the value later, when you will be using it. Please also add it to storybook